src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -729,7 +755,7 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                 var selection = selectionOpt.Value;
 
                 if (workspace.Options.GetOption(EditorComponentOnOffOptions.CodeRefactorings) &&
-                    _owner._codeRefactoringService != null &&
+                    _owner?._codeRefactoringService != null &&
```

Comment:
how can _owner be null?  if this was the case, woudln't we have exploded in the past?